### PR TITLE
Feat/vi colors

### PR DESCRIPTION
**Pull request by jesperwiner**

### CODE REVIEW

File: src/components/alert/alert.jsx [hunk 1]
```diff
@@ -32,12 +32,7 @@ class Alert extends React.PureComponent {
 
     return (
       <button className={this.classes.close} onClick={this.handleCloseClick} type="button">
-        <IconClose
-          stroke={this.theme.palette.text.default}
-          width={10}
-          height={10}
-          style={{ display: 'block' }}
-        />
+        <IconClose stroke="#282823" width={10} height={10} style={{ display: 'block' }} />
```
Comment (KitsuneKelso) [inline src/components/alert/alert.jsx]: Hard coded color here intentional?

File: src/components/editable-input/editable-input.jsx [hunk 2]
```diff
@@ -147,7 +147,7 @@ class EditableInput extends React.Component {
           onClick={this.onSubmit}
           disabled={this.props.disabled || this.props.hasError || this.state.saving}
         >
-          {this.state.saving ? <Spinner size={13} color="#FFF" /> : this.props.submitLabel}
+          {this.state.saving ? <Spinner size={13} color="#FFFFFF" /> : this.props.submitLabel}
```
Comment (KitsuneKelso) [inline src/components/editable-input/editable-input.jsx]: Should we replace this with `theme.palette.white`?

File: src/components/icon/icons/boxCheckmark.jsx [hunk 3]
```diff
@@ -16,13 +16,18 @@ export default function BoxCheckmark({
...
         <path d="M14.5882353,19.0588235 L1.41176471,19.0588235 C0.633411765,19.0588235 0,18.4254118 0,17.6470588 L0,5.41176471 C0,4.63341176 0.633411765,4 1.41176471,4 L14.5882353,4 C15.3665882,4 16,4.63341176 16,5.41176471 L16,17.6470588 C16,18.4254118 15.3665882,19.0588235 14.5882353,19.0588235 Z" />
         <path
           d="M6.20026502,15.00025 C6.14906118,15.00025 6.09785734,14.9806485 6.0586544,14.9418456 L4.05850439,12.9416956 C3.98049854,12.8636898 3.98049854,12.7368803 4.05850439,12.6584744 C4.13651024,12.5808686 4.26331975,12.5808686 4.3413256,12.6584744 L6.19986499,14.5174138 L11.6586744,9.05820437 C11.7366803,8.98059854 11.8634898,8.98059854 11.9414956,9.05820437 C12.0195015,9.13661025 12.0195015,9.26341976 11.9414956,9.34142561 L6.34107558,14.9418456 C6.30187264,14.9810486 6.2506688,15.00025 6.19946496,15.00025 L6.20026502,15.00025 Z"
-          stroke={invert ? fill : '#FFF'}
+          stroke={invert ? fill : '#FFFFFF'}
```
Comment (KitsuneKelso) [inline src/components/icon/icons/boxCheckmark.jsx]: Same here as in editable-input, should both the two instances in here use the color from the theme instead?

File: src/components/icon/icons/boxPlus.jsx [hunk 4]
```diff
@@ -16,14 +16,19 @@ export default function BoxPlus({
...
         <path d="M14.5882353,2.82352941 L1.41176471,2.82352941 C1.152,2.82352941 0.941176471,2.61270588 0.941176471,2.35294118 C0.941176471,2.09317647 1.152,1.88235294 1.41176471,1.88235294 L14.5882353,1.88235294 C14.848,1.88235294 15.0588235,2.09317647 15.0588235,2.35294118 C15.0588235,2.61270588 14.848,2.82352941 14.5882353,2.82352941" />
         <path d="M14.5882353,19.0588235 L1.41176471,19.0588235 C0.633411765,19.0588235 0,18.4254118 0,17.6470588 L0,5.41176471 C0,4.63341176 0.633411765,4 1.41176471,4 L14.5882353,4 C15.3665882,4 16,4.63341176 16,5.41176471 L16,17.6470588 C16,18.4254118 15.3665882,19.0588235 14.5882353,19.0588235 Z" />
         <path
           d="M24 10h-10v-10h-4v10h-10v4h10v10h4v-10h10z"
-          fill={invert ? fill : '#FFF'}
-          stroke={invert ? fill : '#FFF'}
+          fill={invert ? fill : '#FFFFFF'}
+          stroke={invert ? fill : '#FFFFFF'}
```
Comment (KitsuneKelso) [inline src/components/icon/icons/boxPlus.jsx]: ...and same here as above. How fine are we with hard coded colors in here?

File: src/components/spinner/spinner.jsx [hunk 5]
```diff
@@ -63,7 +63,7 @@ function Spinner({
   sheet, // eslint-disable-line react/prop-types
   ...rest
 }) {
-  const usedColor = color || theme.palette.variant.primary;
+  const usedColor = color || '#0046FF';
```
Comment (KitsuneKelso) [inline src/components/spinner/spinner.jsx]: So here we're changing from a palette color to a hard coded one? I'm just gonna stop commenting on these and assume this is fine.

File: src/components/icon/icon.md [hunk 6]
```diff
@@ -1,18 +1,21 @@
 Basic usage:
 
 ```html
-<Icon.Trash fill="tomato" stroke="tomato" width={48} height={48} />
+<Icon.Trash fill="tomato" stroke="tomato" width="{48}" height="{48}" />
```
Comment (anneback) [inline src/components/icon/icon.md]: There should not be double quotation marks here, around `{48}`?

Comment (anneback) [reply src/components/icon/icon.md]: Is it because of the `html` type of the verbatim?

File: src/components/input/help-text.jsx [hunk 7]
```diff
@@ -11,12 +11,13 @@ const styles = theme => {
...
-      color: palette.text.muted,
+      bottom: '-16px',
+      color: palette.gray0,
       ...theme.typography.tertiary(),
     },
-    success: { color: palette.variant.success },
-    warning: { color: palette.variant.warning },
-    error: { color: palette.variant.danger },
+    success: { color: palette.positive },
+    warning: { color: palette.negative },
```
Comment (anneback) [inline src/components/input/help-text.jsx]: `palette.index`

File: src/components/input/input-default-styles.jsx [hunk 8]
```diff
@@ -54,54 +54,54 @@ export default theme => {
...
     input: {
       ...mixins.basicBoxSizing,
-      color: palette.text.default,
+      color: palette.gray0,
       marginBottom: ({ label }) => bottomBorderSize - borderSize + (label ? inputMarginBottom : 0),
       position: 'relative',
-      '&.input--has-success': modifierFn(palette.variant.success),
-      '&.input--has-warning': modifierFn(palette.variant.warning),
+      '&.input--has-success': modifierFn(palette.positive),
+      '&.input--has-warning': modifierFn(palette.negative),
```
Comment (anneback) [inline src/components/input/input-default-styles.jsx]: `palette.index`

File: src/components/progress-bar/progress-bar-styles.jsx [hunk 9]
```diff
@@ -146,45 +123,23 @@ export default theme => {
 
     primary: {
       ...variantModifierFn('primary', {
-        background: palette.background.secondary,
-        backgroundDisabled: palette.background.muted,
-        borderColor: palette.background.secondary,
-        color: palette.shades.dark.text.default,
-        colorDisabled: palette.action.disabled,
+        background: palette.gray7,
```
Comment (anneback) [inline src/components/progress-bar/progress-bar-styles.jsx]: The colors for this segment L126-130 doesn't follow the same pattern as before. Is this intended?

File: src/components/tabs/tabs-styles.jsx [hunk 10]
```diff
@@ -1,12 +1,13 @@
 export default ({ palette }) => ({
   list: {
+    background: 'white',
```
Comment (anneback) [inline src/components/tabs/tabs-styles.jsx]: Not hexvalue here?

File: src/styles/color.js [hunk 11]
```diff
@@ -1,23 +1,37 @@
...
-  blueLight: '#E3F2FC', // Background color in alerts
-  green: '#26A699', // Plus performance, success
-  red: '#FE6F67', // Minus performance, error
-  redLight: '#FE6F67', // Sell
+  // GRAYSCALE PALETTE
 
-  yellow: '#FFD75A', // Notification Logo
-  yellowLight: '#FFF1DD', // Notification
-  white: '#FFF', // Background
+  black: '#000000',
```
Comment (anneback) [inline src/styles/color.js]: Small issue that this isn't sorted like the confluence page.
Black #000000
Gray0 #282823
Gray1 #4B4B46
Gray2 #6E6E69
Gray3 #A0A09B
Gray4 #BCBCB6
Gray5 #D7D7D2
Gray6 #EBEBE8
Gray7 #F5F5F5
White #FFFFFF

File: src/styles/color.js [hunk 12]
```diff
@@ -1,23 +1,37 @@
...
+  gray4: '#BCBCB6',
+  gray3: '#A0A09B',
+  gray2: '#6E6E69',
+  gray1: '#4B4B46',
+  gray0: '#282823',
+  gray7: '#F5F5F5',
+
+  // FUNCTIONAL COLOURS FOR PRODUCT DEVELOPMENT
+
+  index: '#FFCF00',
```
Comment (anneback) [inline src/styles/color.js]: This isn't also in the same order as the confluence page.
CTA #0046FF
Positive #00D200
Negative #FF1900
Index #FFCF00

File: src/components/editable-input/editable-input.jsx (hunk 2 above)
Comment (jesperwiner) [reply src/components/editable-input/editable-input.jsx]: will be fixed

File: src/components/alert/alert.jsx (hunk 1 above)
Comment (jesperwiner) [reply src/components/alert/alert.jsx]: intentional

File: src/components/icon/icons/boxCheckmark.jsx (hunk 3 above)
Comment (jesperwiner) [reply src/components/icon/icons/boxCheckmark.jsx]: we dont have theme here

File: src/components/icon/icons/boxPlus.jsx (hunk 4 above)
Comment (jesperwiner) [reply src/components/icon/icons/boxPlus.jsx]: no theme on icons

File: src/components/input/help-text.jsx (hunk 7 above)
Comment (jesperwiner) [reply src/components/input/help-text.jsx]: we don't have warnings as separate color any more, warnings and errors are both red

File: src/components/input/input-default-styles.jsx (hunk 8 above)
Comment (jesperwiner) [reply src/components/input/input-default-styles.jsx]: no more index, just red for warnings and error

File: src/components/progress-bar/progress-bar-styles.jsx (hunk 9 above)
Comment (jesperwiner) [reply src/components/progress-bar/progress-bar-styles.jsx]: we spread the previous action one as default to minimize the code changes, there is no "grey" default color, same for buttons the first one is a CTA blue.

File: src/components/spinner/spinner.jsx (hunk 5 above)
Comment (jesperwiner) [reply src/components/spinner/spinner.jsx]: good catch, should be CTA, fixing

File: src/components/icon/icon.md (hunk 6 above)
Comment (jesperwiner) [reply src/components/icon/icon.md]: this was just prettier doing whatever prettier does...

File: src/components/tabs/tabs-styles.jsx (hunk 10 above)
Comment (jesperwiner) [reply src/components/tabs/tabs-styles.jsx]: fixing

File: src/styles/color.js (hunk 11 above)
Comment (jesperwiner) [reply src/styles/color.js]: will fix

File: src/styles/color.js (hunk 12 above)
Comment (jesperwiner) [reply src/styles/color.js]: will fix

File: src/components/icon/icons/boxCheckmark.jsx (hunk 3 above)
Comment (JacobBlomgren) [reply src/components/icon/icons/boxCheckmark.jsx]: @jesperwiner [`withTheme` docs](https://github.com/cssinjs/theming#withthemecomponent) 😄

**Pull request merged**